### PR TITLE
Implemented the automatically creation of a database if not exist one; Created a extension class to replace System.Console.WriteLine

### PR DESCRIPTION
**Pull request by jr-araujo**
Hi All,
Here I did some changes in the documentation of ** Lessons 7 **. The changes that were put into this part bring more readability and boost the running of the project created. Why? Specially in ** DocumentStoreHolder ** were made improvements that will automatically create one database for ones that decide to practice this part of Lesson among the all lessons. 

The another one change was to create a class extension to replace the need to repeat `System.Console.WriteLine` throughout the code, bringing more readability.

### CODE REVIEW

File: src/unit1/lesson7/README.md [hunk 1]
```diff
@@ -137,11 +136,25 @@ namespace ContactsManager
...
+                    Urls = new[] { "http://localhost:8080" },
                     Database = "ContactsManager"
                 };
 
-                return store.Initialize();
+                store.Initialize();
+
+                // Gets all database names existing in RavenDB Server
+                var databaseNames =
+                    store.Maintenance.Server.Send(new GetDatabaseNamesOperation(0, 500));
```
Comment (ayende) [inline src/unit1/lesson7/README.md]: Better to use `GetDatabaseRecordOperation`, instead, and check for `null`

File: src/unit1/lesson7/README.md [hunk 2]
```diff
@@ -191,7 +204,27 @@ public class Contact
...
+````csharp
+using static System.Console;
+````
+
+````csharp
+public static class StringExtensions
+{
+    public static void Display(this string value)
+    {
+        WriteLine(value);
```
Comment (ayende) [inline src/unit1/lesson7/README.md]: Better to use `Console.WriteLine` here.

File: src/unit1/lesson7/README.md [hunk 3]
```diff
@@ -231,31 +265,34 @@ private void Run()
...
+### Step 7: Implementing the logic to create a new contact
 
 ````csharp
 private void CreateContact()
 {
     using (var session = DocumentStoreHolder.Store.OpenSession())
     {
-        Console.WriteLine("Name: ");
-        var name = Console.ReadLine();
+        "Name: ".Display();
```
Comment (ayende) [inline src/unit1/lesson7/README.md]: I don't like this. `Console.WriteLine` is something that the reader is familiar with.
The aesthetics to be applied here call for familiarity and lack of surprise above all else.

**Review by ayende — CHANGES_REQUESTED** [review]

File: src/unit1/lesson7/README.md (hunk 3 above)
Comment (jr-araujo) [reply src/unit1/lesson7/README.md]: @ayende 
So, If the better to be used here is to keep `Console.WriteLine` for better readability to anyone reader, I think that class extension doesn't make sense to keep existing anymore, right?

Comment (ayende) [reply src/unit1/lesson7/README.md]: Yes

File: src/unit1/lesson7/README.md (hunk 1 above)
Comment (jr-araujo) [reply src/unit1/lesson7/README.md]: This suggestion was was implemented!!!
😃

File: src/unit1/lesson7/README.md (hunk 2 above)
Comment (jr-araujo) [reply src/unit1/lesson7/README.md]: This class was deleted.

File: src/unit1/lesson7/README.md (hunk 3 above)
Comment (jr-araujo) [reply src/unit1/lesson7/README.md]: It was rolled back to the original implementation using Console.WriteLine

**Review by ayende — APPROVED** [review]

**Pull request merged**